src/routes/portfolio/index.svelte

```diff
@@ -0,0 +1,85 @@
+<main>
```

Comment:
Als je dit erbij doet, wordt de pagina al geprerenderd tijdens de build. Dat is wat beter omdat dit een statische pagina is:
```svelte
<script context="module">
	export const prerender = true;
</script>
```

Reply:
Je kan dan in de service worker ook dit aanvullen: 

`const ssrPages = ['/', '/spellen/offline', '/gymles', '/spellen', '/inloggen', /portfolio];`